src/java.base/linux/native/libsimdsort/avxsort_linux_x86.cpp

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I think this file name should follow pattern of other files: `avx512-linux-qsort.cpp

Reply:
Changed the name of the file as suggested. Please see the latest commit that was pushed.